spec/noughts_and_crosses_spec.rb

```diff
@@ -65,6 +69,20 @@
 
         expect(game.winner).to eq nil
       end
+
+      it "is full when all spots have been taken" do
```

Comment:
When testing specific functions we tend to wrap those tests in a `describe` block, if the function is an instance method like these (i.e. not defined with `def self.foo` but just `def foo`) then the name of that describe block is conventionally `#foo`, if it is a static method defined with `self.`, then it's usually a describe block with `.foo`.